src/dodal/devices/areadetector/andor2.py

```diff
@@ -0,0 +1,47 @@
+from bluesky.protocols import Hints
+from ophyd_async.core import PathProvider, StandardDetector
+from ophyd_async.epics.adcore import ADBaseDatasetDescriber, ADHDFWriter, NDFileHDFIO
+
+from dodal.devices.areadetector.andor2_epics import Andor2Controller, Andor2DriverIO
+
+
+class Andor2(StandardDetector):
+    """
+    Andor 2 area detector device. Andor model:DU897_BV
+
+    Parameters
+    ----------
+    prefix: str
+        Epic Pv,
+    path_provider: PathProvider
+        Path provider for hdf writer
+    name: str
+        Name of the device
```

Comment:
Should: This docstring is on the class but is talking about the params the init. From https://peps.python.org/pep-0257 `The class constructor should be documented in the docstring for its __init__ method. `